about.html

```diff
@@ -79,7 +82,7 @@ <h2>Digitalization In Medicine Forum Logo</h2>
 
         <div id="div31">
```

Comment:
- [ ] **As shown in the original design, the past medicine section should adopt a vertical layout when on mobile versions. Please implement it**


Original Version|Cloned Version
:--------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/125165782-1fdff300-e1b6-11eb-8c3c-0b997b976108.png)|![image](https://user-images.githubusercontent.com/59168430/125165805-32f2c300-e1b6-11eb-8078-92cd9631d8e4.png)

As you can see on mobile sizes, the image cards are vertically in the original design. Please implement this.